### DISCUSSION TITLE
Could you please release the library?

### DISCUSSION THREAD

**Discussion by artem-ag** [graphql-java-kickstart/graphql-java-servlet#302] [Q&A] [answered]
Looks like the last release did not include https://github.com/graphql-java-kickstart/graphql-java-servlet/commit/6a52a3849c26ef923fca40b60901496a274171e6. Could you please make a new release?

**Comment by oliemansm** [comment, marked as answer]
@artem-ag It was released shortly after this message, but there was a problem with syncing to Maven Central. I see it was able to sync in the end.
